src/Entity/Game.php

```diff
@@ -241,6 +279,8 @@ public function replacePuck(string $color)
             $this->remainingTurns = 2;
             $this->switchPlayers();
         }
+
+        $this->graph = clone $this->graph;
```

Comment:
Why ?

Reply:
If I don't do this, Doctrine consider that it has not changes. as it is a reference to an Object instance. It's more a conceptual mistake from me, as I choosed to keep the game state in the board (the graph in fact). It's a mistake. I'm currently changing this. The game state will only be the pucks, current player, and fallenPucks. Do you want to wait for this change to approve this PR?